packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -1,66 +1,93 @@
-<h1 data-translate="datacenter_backup_title"></h1>
+<h1 data-translate="dedicatedCloud_datacenter_backup_title"></h1>
 <div class="datacenter-backup" data-ui-view>
     <div data-ovh-alert="{{ $ctrl.alertMain }}" id="backupMainAlert"></div>
-    <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+    <oui-message
+        class="mb-3"
+        data-ng-if="!$ctrl.backup.isActive()"
+        data-type="info"
+    >
+        <span
+            data-translate="{{ 'dedicatedCloud_datacenter_backup_state_' + $ctrl.backup.state }}"
+        ></span>
+        <span>.&nbsp;</span>
```

Comment:
i'm not sure it's a good idea to merge sentences that way. Maybe use translate-values instead ?